src/js/components/Select/index.d.ts

```diff
@@ -7,14 +7,14 @@ import {
   MarginType,
   PlaceHolderType,
 } from '../../utils';
-
+type disabledKeyType = string | undefined | object | Function;
```

Comment:
why did we include `object` here?

Reply:
In the File \src\js\components\Select\utils.js, we have a function applyKey in which the second argument we are passing is the disabledkey and there we are checking if the typeof Key === 'object'. That is why I have added a type of object.

Reply:
@britt6612 , Can you please reopen this?